### PR TITLE
Seo updates

### PR DESCRIPTION
**Pull request by bkfrye**
# Why
- Needed to correct some syntax for SEO optimization

# What Changed
- Replaced header logo with img tag instead of `<h1>` with background img
- Switched `<h2>` with `<h1>` on product pages
-- Should promote better rankings for products

### CODE REVIEW

**Comment by bkfrye** [comment]
I only made CSS changes to the header logo, so it shouldn't change anything else on the site. Let me know if you see any issues that cascade down.

File: wp-content/themes/great-outdoor/header.php [hunk 1]
```diff
@@ -39,7 +39,9 @@
 
         <header class="header">
 
-            <a href="<?php echo esc_url( bloginfo('url') ); ?>"><h1><?php esc_html( bloginfo('name') ); ?></h1></a>
+            <a href="<?php echo esc_url( bloginfo('url') ); ?>">
+                <img src="<?php echo get_stylesheet_directory_uri() . '/assets/logo.png' ?>" alt="GOPC logo">
```
Comment (joshmobley) [inline wp-content/themes/great-outdoor/header.php]: I'm wondering if we should use `.logo` or some other class here to add specificity to this particular image.  I can think of a few places in the mega-menus where images are being used, and `header img` would likely cascade down. Alternatively perhaps we just style it with `header > img`?

**Review by joshmobley — CHANGES_REQUESTED** [review]
Wondering about CSS specificity, take a look at my comment.

File: wp-content/themes/great-outdoor/header.php (hunk 1 above)
Comment (bkfrye) [reply wp-content/themes/great-outdoor/header.php]: To keep things explicit, I just added an id to the image. It probably doesn't need to be nested like `header #masthead` but I went with it. Thanks for the feedback to help things move along more smoothly.

**Review by joshmobley — APPROVED** [review]
thanks!

**Pull request merged**